internal/jimm/model.go

```diff
@@ -748,16 +749,99 @@ func (j *JIMM) ModelInfo(ctx context.Context, user *openfga.User, mt names.Model
 	return j.mergeModelInfo(ctx, user, mi, m)
 }
 
+type modelSummariesSafeMap struct {
```

Comment:
I suggest dropping `safe` in the name. Also add a docstring describing what they key represents or uses a better type than string if possible.
```suggestion
type modelSummariesMap struct {
```